src/device/device-management.ts

```diff
@@ -101,15 +109,14 @@ export async function createDisplayTrack(): Promise<LocalDisplayTrack> {
  * @returns List of media devices in an array of MediaDeviceInfo objects.
  */
 export async function getDevices(deviceKind?: media.DeviceKind): Promise<MediaDeviceInfo[]> {
-  let devices: MediaDeviceInfo[];
-  try {
-    devices = await media.ensureDevicePermissions(
+  const devices: MediaDeviceInfo[] = await media
+    .ensureDevicePermissions(
       [media.DeviceKind.AudioInput, media.DeviceKind.VideoInput],
       media.enumerateDevices
-    );
-  } catch (error) {
-    throw new WcmeError(ErrorTypes.DEVICE_PERMISSION_DENIED, 'Failed to ensure device permissions');
-  }
+    )
+    .catch((error) => {
+      throw error;
```

Comment:
what's the point of this catch here now? I think it can be removed

Reply:
done